libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -992,12 +1034,28 @@ void AP_MotorsUGV::output_throttle(SRV_Channel::Function function, float throttl
         return;
     }
 
+    // get dt from scheduler if not provided (dt <= 0)
+    // fallback to 50Hz if scheduler rate unavailable
+    if (dt <= 0.0f) {
+        dt = 0.02f;
+        const uint16_t loop_hz = AP::scheduler().get_loop_rate_hz();
+        if (loop_hz > 0) {
+            dt = 1.0f / float(loop_hz);
+        }
+    }
```

Comment:
Why is this needed? We don't need it for the existing users of dt, they just have a `is_positive` check.

Reply:
Did you see issues before you added the check? Or are you just guessing that there may be a issue?